source/js/main.js

```diff
@@ -222,31 +215,26 @@ let main = {
       });
     }
 
-    let donateFooterBtn = document.getElementById(`donate-footer-btn`);
-    if (donateFooterBtn) {
-      donateFooterBtn.addEventListener(`click`, () => {
-        ReactGA.event({
-          category: `donate`,
-          action: `donate button tap`,
-          label: `${document.title} footer`
-        });
-      });
-    }
+    common.bindEventHandlers();
   },
 
   bindGAEventTrackers() {
     let seeMorePage = document.querySelector(`#see-more-modular-page`);
 
     if (seeMorePage) {
       seeMorePage.addEventListener(`click`, () => {
-        let label = ``;
-        let pageHeader = document.querySelector(`.cms h1`);
-
-        if (pageHeader) {
-          label = `${pageHeader.innerText} - footer cta`;
-        }
+        let label =
```

Comment:
this is less readable than before, and now calls `document.querySelectorAll` twice, so unless there was a good reason to rewrite it, the old code is actually better.